README.md

```diff
@@ -35,9 +35,7 @@ If you're ready to contribute and create your PR, it will help to set up a local
 
 1. [Set up your development environment](https://www.gatsbyjs.com/docs/tutorial/part-zero/)
 
-2. [Fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) this repo
```

Comment:
![image](https://user-images.githubusercontent.com/54227730/104778909-fed0a500-5732-11eb-95fa-b924b9850a08.png)

This step is already declared above this prior to local setup as a necessary step, even if you don't chose to work locally. 

Alternatively we could just get rid of the "Fork the repository(repo)" section and just leave it in the "Set up your local environment (optional)" section. I did it this way though since some people may just want to fork it on GitHub and edit from the web interface without cloning and setting up locally.